applications/nrf5340_audio/README.rst

```diff
@@ -664,7 +664,10 @@ The application uses the following buttons on the supported development kit:
 |               | * Pressed on the gateway during playback multiple times: Changes the tone frequency.   |
 |               |   The available values are 1000 Hz, 2000 Hz, and 4000 Hz.                              |
 +---------------+----------------------------------------------------------------------------------------+
-| **BTN 5**     | Mutes the playback volume.                                                             |
+| **BTN 5**     | Depending on the moment it is pressed:                                                 |
+|               |                                                                                        |
+|               | * Long-pressed during startup: Clear previously stored bonding information             |
```

Comment:
Need a period at the end of the sentence

Reply:
Done